### PR TITLE
Block non-ckan routes

### PR DESCRIPTION
**Pull request by robert-bryson**
# Pull Request

Related to https://github.com/GSA/data.gov/issues/4044

## PR TASKS

- [x] The actual code changes.
- ~~[ ] Tests written and passed.~~
- [ ] Any changes to docs?
- ~~[ ] Any new
[requirements versions](https://github.com/GSA/inventory-app/blob/main/requirements.in.txt)
to pull in?~~

### CODE REVIEW

**Comment by nickumia-reisys** [comment]
Login is broken....

File: proxy/nginx.conf [hunk 1]
```diff
@@ -38,10 +38,18 @@ http {
     # path for static files
     root ./public;
 
-    location / {
+    # disallow any routes but below
+    location / { deny all; }
+
+    # allow literal /
+    location = / { try_files $uri @proxy_to_app; }
+    location ~* ^/(stats|organization|feeds|ckan-admin|about|api|group|webassets|dashboard|user|fonts|base|acs|slo|datastore|dataset|uploads|testing|public|organization|images|solr)(?:/(.*))?$ {
```
Comment (nickumia-reisys) [inline proxy/nginx.conf:46]: I don't see `acs` and `slo` allowed here.

**Comment by robert-bryson** [comment]
> Login is broken....
> 
> Edit: Unless it's just my account + firefox with the issue.

Hmm, it is working for me in new Firefox/Chrome windows?

**Review by nickumia-reisys — APPROVED** [review]
🍝

**Pull request merged**